src/momento/internal/aio/_vector_index_grpc_manager.py

```diff
@@ -19,7 +22,7 @@ class _VectorIndexControlGrpcManager:
     def __init__(self, configuration: VectorIndexConfiguration, credential_provider: CredentialProvider):
         self._secure_channel = grpc.aio.secure_channel(
             target=credential_provider.control_endpoint,
-            credentials=grpc.ssl_channel_credentials(),
+            credentials=vector_credentials_from_root_certs_or_default(configuration),
```

Comment:
so this is what you were saying about this will only work with vector for now?

i'd be fine with you making that util function general and using in other clients but it's not a blokcer for this pr

Reply:
Yes that's right. Can open up in a follow up PR.